src/utils/api/getEmployees.ts

```diff
@@ -1,5 +1,5 @@
 const externalGetEmployeesURL =
-  'https://variant-homepage-proxy.azurewebsites.net/api/getEmployees?code=rM%2FOUPE7QtnGa6el%2FAro%2FT2u7Zc029LU2CaOtlbn6lz4l3UoceauVw%3D%3D';
+  'https://variant-homepage-proxy.azurewebsites.net/api/getEmployees';
```

Comment:
Is this a regression error now? cc @MathiasFl

Reply:
Ahh yeah there was some issue while testing. But it might need to be put back before merge

Reply:
Apparently tries Azure to authenticate you with passed key even if auth is set to "anonymous", so when the missing app keys are happening it's not possible to test. This was temp hotfix, should be removed from commit or get updated by rebasing onto master before merge.